tests/data/region_processing/dsd/variables/tag_fuel.yaml

```diff
@@ -0,0 +1,3 @@
+- <Fuel>:
```

Comment:
I would not use a `DataStructureDefintion` with a tag here - the test should be as focused as possible.

Reply:
Done